pkg/guacrest/server/retrieveDependencies_test.go

```diff
@@ -482,151 +480,41 @@ func Test_RetrieveDependencies(t *testing.T) {
 			gqlClient := SetupTest(t)
 			Ingest(ctx, t, gqlClient, tt.data)
 
-			restApi := server.NewDefaultServer(gqlClient)
+			encodedPurl := url.QueryEscape(tt.purl)
+			encodedArtifact := url.QueryEscape(tt.digest)
 
-			/******** call the endpoint with byName link condition ********/
-			inputByName := tt.input
-			inputByName.LinkCondition = ptrfrom.Any(api.Name)
-			resByName, err := restApi.RetrieveDependencies(ctx, api.RetrieveDependenciesRequestObject{Params: inputByName})
-			if err != nil {
-				t.Fatalf("Endpoint returned unexpected error: %v", err)
-			}
-			/******** check the output ********/
-			switch v := resByName.(type) {
-			case api.RetrieveDependencies200JSONResponse:
-				if !cmp.Equal(v.PurlList, tt.expectedByName, cmpopts.EquateEmpty(), cmpopts.SortSlices(stdcmp.Less[string])) {
-					t.Errorf("RetrieveDependencies with byName returned %v, but wanted %v", v.PurlList, tt.expectedByName)
+			if tt.purl != "" {
+				/******** call the endpoint with byName link condition ********/
+				resByName, err := server.GetDepsForPackage(ctx, gqlClient, encodedPurl)
+				if err != nil {
+					t.Fatalf("Endpoint returned unexpected error: %v", err)
 				}
-			default:
-				t.Errorf("RetrieveDependencies with byName returned unexpected error: %v", v)
-			}
-
-			/******** call the endpoint with byDigest link condition ********/
-			inputByDigest := tt.input
-			inputByDigest.LinkCondition = ptrfrom.Any(api.Digest)
-			resByDigest, err := restApi.RetrieveDependencies(ctx, api.RetrieveDependenciesRequestObject{Params: inputByDigest})
-			if err != nil {
-				t.Fatalf("Endpoint returned unexpected error: %v", err)
-			}
-			/******** check the output ********/
-			switch v := resByDigest.(type) {
-			case api.RetrieveDependencies200JSONResponse:
-				if !cmp.Equal(v.PurlList, tt.expectedByDigest, cmpopts.EquateEmpty(), cmpopts.SortSlices(stdcmp.Less[string])) {
-					t.Errorf("RetrieveDependencies with byDigest returned %v, but wanted %v", v.PurlList, tt.expectedByDigest)
+				/******** check the output ********/
+				var purls []string
+				for _, purl := range resByName {
+					purls = append(purls, purl)
+				}
+				if !cmp.Equal(purls, tt.expectedByName, cmpopts.EquateEmpty(), cmpopts.SortSlices(stdcmp.Less[string])) {
+					t.Errorf("RetrieveDependencies with byName returned %v, but wanted %v", purls, tt.expectedByName)
 				}
-			default:
-				t.Errorf("RetrieveDependencies with byDigest returned unexpected error: %v", v)
-			}
-		})
-	}
-}
-
-func Test_ClientErrors(t *testing.T) {
```

Comment:
Any reason that these tests were removed?

Reply:
My bad, I removed these tests to split them into two different ones for purls and digests, but forgot to add them back in.

I have added the test back (The test is split into two test functions, one for purls and one for digests).